src/EventListener/SerializeListener.php

```diff
@@ -60,7 +60,9 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
 
         $context = $this->serializerContextBuilder->createFromRequest($request, true, $attributes);
         $resources = [];
-        $context['resources'] = &$resources;
+        $context['resources'] = function ($iri) use (&$resources) {
```

Comment:
Yes but it's fucked up that a Closure is faster then a reference in this case. There may be a simple explanation with the internal memory pointer doing less work, I would love to understand :p.